marimbabot_command_recognition/src/text2speech_client.py

```diff
@@ -0,0 +1,13 @@
+from marimbabot_command_recognition.srv import Speak, SpeakResponse
+import rospy
+
+# only for show how to call the service of text2speech service
```

Comment:
could you move it into an examples or scripts folder, so it is clear what script is for testing/demonstrations

Reply:
done